neutron/assetlist.json

```diff
@@ -446,6 +446,34 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/babycorgi.png"
         }
       ]
+    },
+    {
+      "description": "Jimmy Neutron Finance",
+      "denom_units": [
+        {
+          "denom": "factory/neutron108x7vp9zv22d6wxrs9as8dshd3pd5vsga463yd/JIMMY",
+          "exponent": 0,
+          "aliases": [
+            "ujimmy"
+          ]
+        },
+        {
+          "denom": "jimmy",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/neutron108x7vp9zv22d6wxrs9as8dshd3pd5vsga463yd/JIMMY",
+      "name": "jimmy",
+      "display": "jimmy",
+      "symbol": "JIMMY",
+      "logo_URIs": {
+        "png": "https://github.com/JIMMwe/chain-registry/blob/master/neutron/images/jimmy.png"
```

Comment:
The image URLs must point to this org and repo, so something like:
https://raw.githubusercontent.com/cosmos/chain-registry/master/neutron/images/jimmy.png

Reply:
Although the URL I comment isn't valid yet, it will be once this is merged. Just update the URLs now